cmd/flags_network.go

```diff
@@ -85,6 +85,11 @@ var (
 		Name:  "experiment-natpunching",
 		Usage: "Enables experimental NAT hole punching",
 	})
+
+	enableDNSFlag = altsrc.NewBoolTFlag(cli.BoolTFlag{
```

Comment:
As it's connection choice, the best would be to have it as option passed to connection creation, instead global flag

Reply:
Moved to connection options.